app/packs/src/components/ElementsTableSettings.js

```diff
@@ -140,9 +162,17 @@ export default class ElementsTableSettings extends React.Component {
             </FormGroup>
             <FormGroup>
               <Checkbox onChange={this.handleToggleSampleExt}
-                        checked={showSampleExternalName} >
+                        checked={showSampleExternalLabel} >
                 Show sample external name on title
               </Checkbox>
+              <Checkbox onChange={this.handleToggleSampleShortLabel}
+                        checked={showSampleShortLabel} >
```

Comment:
Indentation issue and the closing bracket must be aligned with the line containing the opening tag